### PR TITLE
Updates to Fix plan shopping display bug

### PR DESCRIPTION
**Pull request by jayreddy519**
[RM-97731](https://devops.dchbx.org/redmine/issues/97731)

### CODE REVIEW

File: app/views/insured/plan_shoppings/show.html.slim [hunk 1]
```diff
@@ -348,7 +348,7 @@ javascript:
         networkTypes.forEach(function (networkType) {
           dataArray.forEach(function (enrollment) {
             if (networkType.toLowerCase() == 'dc-metro' || networkType.toLowerCase() == 'maine') {
-              if (enrollment.product.in_state_network) {
+              if (enrollment.product.dc_in_network) {
```
Comment (markgoho) [inline app/views/insured/plan_shoppings/show.html.slim:351]: can we avoid referencing specific states?

File: features/support/object_model_pages/shop/employee/registration/ee_choose_plan.rb [hunk 2]
```diff
@@ -39,11 +39,11 @@ def self.pos_checkbox
     'label[for="checkbox-12"]'
   end
 
-  def self.dc_metro
+  def self.nationwide
     'label[for="checkbox-15"]'
   end
 
-  def self.nationwide
+  def self.dc_metro
```
Comment (markgoho) [inline features/support/object_model_pages/shop/employee/registration/ee_choose_plan.rb:46]: let's avoid referencing a client here as well

File: system/config/templates/features/enroll_app/enroll_app.yml [hunk 3]
```diff
@@ -161,6 +161,8 @@ registry:
             item: "https://staging.dchealthlink.com/individuals/tax-documents"
           - key: :statewide_area
             item: "DC-Metro"
+          - key: :nationwide_area
```
Comment (markgoho) [inline system/config/templates/features/enroll_app/enroll_app.yml:164]: what is the purpose of this setting? does it change between clients?

**Review by markgoho — CHANGES_REQUESTED** [review]

File: system/config/templates/features/enroll_app/enroll_app.yml (hunk 3 above)
Comment (jayreddy519) [reply system/config/templates/features/enroll_app/enroll_app.yml:164]: network information related to DC plans. its specific to DC only.

File: app/views/insured/plan_shoppings/show.html.slim (hunk 1 above)
Comment (jayreddy519) [reply app/views/insured/plan_shoppings/show.html.slim:351]: dc_in_network is a field under components/benefit_markets/app/models/benefit_markets/products/product.rb.
& we are using that particular field under plan shopping page.

NOTE: It is being used  in many different places around the application, May be we have to create a new task to refactor state based references.

**Review by markgoho — APPROVED** [review]

**Review by RyanEddyIC — APPROVED** [review]

**Pull request merged**